featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -715,12 +735,32 @@ def test_update_dataframe_dont_recalculate_last_time_index(es):
 
     new_dataframe = es['customers'].copy()
     new_dataframe['signup_date'] = new_time_index
+    new_dataframe.pop('_ft_last_time')
 
     es.update_dataframe('customers', new_dataframe, recalculate_last_time_indexes=False)
-    pd.testing.assert_series_equal(to_pandas(es['customers']['last_time'], sort_index=True), to_pandas(original_last_time_index, sort_index=True))
+    assert 'last_time_index' not in es['customers'].ww.metadata
+    assert original_lti_name not in es['customers'].columns
+
+
+def test_update_dataframe_recalculate_last_time_index_not_present(es):
+    es.add_last_time_indexes()
+
+    original_time_index = es['log']['datetime'].copy()
+
+    if ks and isinstance(original_time_index, ks.Series):
+        new_time_index = ks.from_pandas(original_time_index.to_pandas() + pd.Timedelta(days=10))
+    else:
+        new_time_index = original_time_index + pd.Timedelta(days=10)
+
+    new_dataframe = es['log'].copy()
+    new_dataframe['datetime'] = new_time_index
+    new_dataframe.pop('_ft_last_time')
+
+    es.update_dataframe('log', new_dataframe, recalculate_last_time_indexes=True)
+    pd.testing.assert_series_equal(to_pandas(es['log']['datetime']).sort_index(), to_pandas(new_time_index).sort_index(), check_names=False)
```

Comment:
Do we also need to verify that the last time index column is now present after the update?

Reply:
yep--the check for whether the lti column matches what we expect accesses `es['log'][LTI_COLUMN_NAME]` so that should cover it!